### ISSUE TITLE
Postman 7.0 not available via snap

### ISSUE THREAD

**Issue by sssurii** [a85/POSTMan-Chrome-Extension#6024]
Please read through the [guidelines](https://github.com/postmanlabs/postman-app-support#guidelines-for-reporting-issues) before creating a new issue.

**Describe the bug**
I am using postman app on ubuntu 16.04, had stop sync process after i accepted the update notification and clicked the button `Migrate my team to v7.0`. I am unable to use postman because of this. 

**To Reproduce**
Sorry can't go back to reproduce it.
1. Login and got the update notification first-time (if using old version).
2. Click on 'Migrate my team to v7.0'
3. Now its showing a warning message: `Your team updated to Postman v7.0. To keep your collections and workspaces in sync with your team, you must also update to v7.0 by going to  > Settings > Update.`
4. Even i can't see any tab or button on mentioned location in above warning message to update v.7.0

**Expected behavior**
Please suggest a way to resolve this issue or to revert my account setting OR postman version to old state so that i can continue my work.

**Screenshots**
![d098kiaucaa6asn](https://user-images.githubusercontent.com/38280750/53880784-ec19bf00-4037-11e9-8b0c-72cb162c39bf.jpg)

**App information:**
 - Native App
 - Postman Version - 6.7.1
 - OS: Ubuntu 16.04

**Comment by ionutale** [comment]
me 2 
but i'm on windows 10

**Comment by numaanashraf** [comment]
@jhonyourangel Could you open another issue for this? This is not related to the original issue.

**Comment by numaanashraf** [comment]
Postman 7.0 is currently on candidate line on Snap.

```
sudo snap switch --channel=candidate postman
sudo snap refresh postman
```

**Comment by gryphon2411** [comment]
Postman released *stable* version 7.6.0 on 29/08.

Why the most recent *stable* version on snap is 6.7.1? doesn't 7.6.0 is stable enough to be on snap *stable* channel rather than candidate channel?

**Comment by PranY** [comment]
The below command (by default) kept me on 6.7
`sudo snap switch --channel=candidate postman`
and the output was 
`"postman" switched to the "v6/candidate" channel`

I forcefully changed the channel to 'latest/stable' so the new command was
`sudo snap switch --channel=latest/stable postman`
and the output was
`"postman" switched to the "latest/stable" channel`

Then updating the snap with `sudo snap refresh postman` works perfectly. I updated from 6.7 to 7.22, hope it helps you.
